relay-server/Cargo.toml

```diff
@@ -17,6 +17,7 @@ processing = [
     "minidump",
     "rdkafka",
     "rdkafka-sys",
+    "rdkafka-sys/cmake-build",
```

Comment:
Do you know if it is possible to activate this only on windows?

Reply:
That would be a nightly feature:
https://github.com/rust-lang/cargo/issues/7914
(https://github.com/rust-lang/cargo/issues/2524)

Reply:
Gotcha, thanks. Let's add a note to the README then (where it says that we require latest rust), then this is good to merge 👍